### PR TITLE
init_project

### PR DESCRIPTION
**Pull request by ngochieu1297**

### CODE REVIEW

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (duccnv-1684) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: enter last line

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (duccnv-1684) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: enter last line

File: app/src/test/java/com/example/music_58/ExampleUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (duccnv-1684) [inline app/src/test/java/com/example/music_58/ExampleUnitTest.java]: enter last line

**Pull request merged**